code/modules/mob/living/living.dm

```diff
@@ -692,10 +694,8 @@
 						if(spec_color)
 							TH.color = spec_color
 
-/mob/living/carbon/human/makeTrail(turf/T, turf/start, direction, spec_color)
-	if((NOBLOOD in dna.species.species_traits) || !bleed_rate || bleedsuppress)
-		return
-	spec_color = dna.species.blood_color
```

Comment:
why is this stuff deleted?

Reply:
it's replaced with this: cause Squishy suggested to do this in order to get the changing blood color to work for E-N 
/mob/living/simple_animal/pet/dog/corgi/borgi/makeTrail(turf/T, turf/start, direction, spec_color)
	spec_color = blood_color
	..()

Reply:
well, wouldn't this break the makeTrail proc for humans then?

Reply:
And also, this has nothing to do with the borgi. Its not even a subtype of human

Reply:
i mean it was DocterSquishy's suggestion i didn't really realize much of it